hazelcast-client/src/main/java/com/hazelcast/client/impl/LifecycleServiceImpl.java

```diff
@@ -80,8 +79,10 @@ public boolean removeLifecycleListener(String registrationId) {
 
     public void fireLifecycleEvent(LifecycleEvent.LifecycleState lifecycleState) {
         final LifecycleEvent lifecycleEvent = new LifecycleEvent(lifecycleState);
-        getLogger().info("HazelcastClient[" + client.getName() + "]" + "["
-                + buildInfo.getVersion() + "] is " + lifecycleEvent.getState());
+        String revision = buildInfo.getRevision();
+        getLogger().info("HazelcastClient " + buildInfo.getVersion() + " ("
+                + buildInfo.getBuild() + (revision != null ? " - " + revision : "") + ") is "
```

Comment:
Output of this line is as follows:
hz.client_0 [dev] [3.7-SNAPSHOT] HazelcastClient 3.7-SNAPSHOT (20160201 - bc4f0b5) 
Where following is from DefaultLogger
hz.client_0 [dev] [3.7-SNAPSHOT]  
And more detailed info that includes buildInfo is from implementation in LifeCycleService 
HazelcastClient 3.7-SNAPSHOT (20160201 - bc4f0b5) 

Only version is duplicate here which was requirement and also in line with server 
Similar line of server is as follows:
INFO: [127.0.0.1]:5701 [dev] [3.7-SNAPSHOT] Hazelcast 3.7-SNAPSHOT (20160216 - bc4f0b5) starting at Address[127.0.0.1]:5701